src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -1308,6 +1391,38 @@ else if (broadcast.getType().equals(AntMediaApplicationAdapter.PLAY_LIST)) {
 		return result;
 	}
 
+	public boolean forwardStartStreaming(Broadcast broadcast) {
+		String jwtToken = JWTFilter.generateJwtToken(getAppSettings().getClusterCommunicationKey(), System.currentTimeMillis() + 5000);
+		String restRouteOfNode = "http://" + broadcast.getOriginAdress() + ":" + getServerSettings().getDefaultHttpPort() +
+				File.separator + getAppSettings().getAppName() + File.separator + "rest" +
+				File.separator + "v2" + File.separator + "broadcasts" +
+				File.separator + broadcast.getStreamId() + File.separator + "start";
+
+		CompletableFuture<Boolean> future = new CompletableFuture<>();
+
+		vertx.executeBlocking(() -> {
+			try {
+				boolean result = sendClusterPost(restRouteOfNode, jwtToken, CLUSTER_POST_RETRY_ATTEMPT);
+				future.complete(result);
+			} catch (Exception exception) {
+				logger.error(ExceptionUtils.getStackTrace(exception));
+				future.complete(false);
+			}
+			return null;
+		}, false);
+
+		try {
+			// Total timeout = retry attempts * (retry delay + request timeout) + buffer
+			long totalTimeout = CLUSTER_POST_RETRY_ATTEMPT * (appSettings.getWebhookRetryDelay() + 1000) + 1000;
+			return future.get(totalTimeout, TimeUnit.MILLISECONDS);
```

Comment:
Second blocking call. Please don't use blocking calls if it's not necessary.

Reply:
done